serverlessgo/main.go

```diff
@@ -105,23 +101,32 @@ func (lb *loadBalancer) Run() {
 			}
 			lb.arrivalCond.Set(false)
 		}
+		lb.tryScaleDown()
```

Comment:
Por que só fazer scale down quando for fazer scale up? Acho que tem que ir para fora do if.

Reply:
Acho que você confundiu. A gente tenta fazer scale down em cada laço do run do load balancer.